src/middlewares/lambdaIsWarmingUp.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (config) => {
+  // get isWarmingUp boolean from config object or use approach with serverless-warmup-plugin library
+  const isWarmingUp = event => (config && config.isWarmingUp) || event.source === 'serverless-warmup-plugin'
```

Comment:
I believe `isWarmingUp` should be a function that accepts `event` as an argument so I would change this part as follows (also following the way of handling default options in other middlewares):

```javascript
const defaults = {
  isWarmingUp: (event) => event.source === 'serverless-warmup-plugin'
}

module.exports = (opts) => ({
  before: (handler, next) => {
    const options = Object.assign({}, defaults, opts)
      if (isWarmingUp(handler.event)) {
        // ...
      }
    
    // ...
  }
})
```

Reply:
I got the idea of passing function instead of boolean into middleware.
That's what I thought when write the issue.
Thanks for suggestion!